yarn.lock

```diff
@@ -15,7 +15,7 @@
     "@jridgewell/gen-mapping" "^0.1.0"
     "@jridgewell/trace-mapping" "^0.3.9"
 
-"@arbitrum/nitro-contracts@1.0.1":
+"@arbitrum/nitro-contracts@1.0.1", "@arbitrum/nitro-contracts@^1.0.0-beta.8":
```

Comment:
why the beta version is used?

Reply:
https://www.npmjs.com/package/@arbitrum/token-bridge-contracts?activeTab=versions

Looks like it's the only version that's been released